ICAFIX/hcp_fix

```diff
@@ -119,31 +124,41 @@ fi
 
 hp=$2
 
-#this case logic could go in a helper function, interpret_as_bool or something
-case $(echo "$3" | tr '[:upper:]' '[:lower:]') in
-    ( true | yes )
-        doMotionRegression=TRUE
+function interpret_as_bool()
+{
+    case $(echo "$1" | tr '[:upper:]' '[:lower:]') in
+    (true | yes | 1)
+        echo TRUE
         ;;
-    ( false | no | none )
-        doMotionRegression=FALSE
-
-		# 5/20/2019: For the time being, do_motion_regression=FALSE doesn't work properly with single-run FIX,
-		# so don't allow that mode of operation
-		# https://github.com/Washington-University/HCPpipelines/issues/108
-		log_Err_Abort "Due to a restriction in the FIX argument parsing, do_motion_regression=$3 is not currently supported"
-
-		;;
-    ( * )
-        log_Err_Abort "error: '$3' is not valid for <do_motion_regression>, please use TRUE or FALSE"
+    (false | no | none | 0)
+        echo FALSE
+        ;;
+    (*)
+        log_Err_Abort "error: '$1' is not valid for this argument, please use TRUE or FALSE"
         ;;
-esac
+    esac
+}
+
+doMotionRegression=$(interpret_as_bool "$3")
+# 5/20/2019: For the time being, do_motion_regression=FALSE doesn't work properly with single-run FIX,
+# so don't allow that mode of operation
+# https://github.com/Washington-University/HCPpipelines/issues/108
+if [[ "$doMotionRegression" == "FALSE" ]]
```

Comment:
Shouldn't the check here be against 0, since it was converted to boolean at L142?

Reply:
We can't be consistent with ReApply without rewriting more of the code, which I don't want to do right before a release.  By the way, both hcp_fix* scripts use TRUE/FALSE.